cli/src/main/kotlin/commands/ClearlyDefinedUploadCommand.kt

```diff
@@ -142,24 +142,26 @@ class ClearlyDefinedUploadCommand : CliktCommand(
             )
         }
 
-        var error = false
+        var uploadedCurationsCount = 0
         val uploadableCurations = curationsByHarvestStatus[HarvestStatus.HARVESTED].orEmpty() +
                 curationsByHarvestStatus[HarvestStatus.PARTIALLY_HARVESTED].orEmpty()
 
         uploadableCurations.forEachIndexed { index, curation ->
             print("Curation ${index + 1} of ${uploadableCurations.size} for package '${curation.id.toCoordinates()}' ")
 
             when (val summary = putCuration(curation)) {
-                null -> {
-                    println("failed to be uploaded.")
-                    error = true
-                }
+                null -> println("failed to be uploaded.")
+                else -> {
+                    println("was uploaded successfully:\n${summary.url}")
 
-                else -> println("was uploaded successfully:\n${summary.url}")
+                    uploadedCurationsCount++
```

Comment:
Nit: Mind changing this to a prefix increment, like `++uploadedCurationsCount`? Before you ask, it's just a matter of personal taste, and I prefer the prefix increment if the return of the previous value is not needed, like here.

Reply:
Done.